### ISSUE TITLE
signup with email returns 404

### ISSUE THREAD

**Issue by valkmit** [Infisical/infisical#913]
### Describe the bug
I am self-hosting Infisical with Kubernetes and have the following configuration settings

```
frontend:
  replicaCount: 1
backend:
  replicaCount: 1
ingress:
  enabled: false
backendEnvironmentVariables:
  INVITE_ONLY_SIGNUP: true
```

Visiting the frontend works fine with no errors. However, when I attempt to register the 1st account on my fresh installation, the API request that hits signup is 404'd.

![image](https://github.com/Infisical/infisical/assets/76670147/113f80e6-59e8-4820-9e4b-620ee838b434)

Below is the chart and release version

```
$ helm list     
NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
secops          default         1               2023-08-29 23:05:59.676102714 -0700 PDT deployed        infisical-0.3.2         1.17.0
```

Thanks in advance!

**Comment by akhilmhdh** [comment]
@dangtony98 anything related to sso changes recently

**Comment by maidul98** [comment]
Do you have `ingress` configured? Infisical needs ingress to properly route traffic

**Comment by CypriotUnknown** [comment]
same here

**Comment by valkmit** [comment]
> Do you have `ingress controller` configured? Infisical needs ingress to properly route traffic

No, I have that disabled as per the config as I am using cloudflared to connect my cluster to the internet.

I have it forwarding all http requests from a subdomain to the frontend service. Is there any way to get it to work without an ingress controller?

**Comment by maidul98** [comment]
The current setup requires Ingress to route traffic from frontend to backend services. The issue you have right now is the frontend is not able to route traffic to the backend services and therefore you get 404.

If you want to route traffic between the backend and frontend using cloudflared, you'll need to configure the equivalent routing with it as here https://github.com/Infisical/infisical/blob/main/helm-charts/infisical/templates/ingress.yaml

**Comment by akhilmhdh** [comment]
Closing this issue as stale. The new charts resolve this as we now have a standalone image.
